src/log/Log.h

```diff
@@ -24,7 +24,7 @@ namespace logging {
 class Graylog;
 class SubsystemMap;
 
-class Log : private Thread
+class Log : public Thread
```

Comment:
we could also just
```
using Thread::is_started();
```
in `Log` 's "public" section to expose only the specific method to its consumers, instead of making `Log` a `Thread`.